docs/source/scores/guidelines.rst

```diff
@@ -5,8 +5,62 @@ DCML Score Conventions
 .. contents:: Contents
    :local:
 
+General Principles
+==================
+
+These guidelines are intended to ensure that the scores produced by and for the DCML are consistent and high-quality
+encodings of some print or manuscript edition. Although they represent digital editions, first and foremost they
```

Comment:
Do you mean "some print or other digital edition"? I don't know what "manuscript edition" means (unless you mean facsimile?).